students/andrew_garcia/lesson_03/Mailroom.py

```diff
@@ -0,0 +1,112 @@
+'''
+Andrew Garcia
+Mailroom Part 1
+6/22/19
+'''
+
+#!/usr/bin/env python3
+
+all_donations = [['Jayson Black', [156.80, 207.32, 219.92]],
+          ['Bryan Fultan', [1236.28]],
+          ['Danica Dolores', [163.51, 100.42, 186.22]],
+          ['Skylar Odell', [167.90, 151.62]],
+          ['Roy Max', [137.97, 227.63]]]
+
+
+def options_menu():
+    answer = input('''
+Options:
+1 - Send a Thank You
+2 - View a Report
+3 - Quit
+    
+    Select an Option: ''')
+
+    if answer == '1':
+        send_thank_you()
+    elif answer == '2':
+        create_report()
+    elif answer == '3':
+        pass
+    else:
+        print('That is not a valid answer!')
+        options_menu()
+
+
+def format_thank_you(name, amount):
+    print(f'''\nDear {name.title()},
+
+Thank you so much for your generous donation of ${amount:.2f}!
+The kind generosity that you and others give us helps us complete the great work we are doing for others.
+
+From,
+Your Local Charity
+\n''')
+    options_menu()
+
```

Comment:
Add another blank row between functions - looks like you're doing that for most of these, but this one got missed?